src/main/kotlin/com/fairchild_superconductor/magic_crystal/MagicCrystal.kt

```diff
@@ -20,34 +32,78 @@ var COMPUTER_ENTITY: BlockEntityType<ComputerEntity>? = null
 object MagicCrystal : ModInitializer {
     const val MOD_ID = "magic_crystal"
     val COMPUTER_BLOCK: ComputerBlock = ComputerBlock(FabricBlockSettings.of(Material.STONE).hardness(4.0f))
+    val TIN_ORE_BLOCK: Block = Block(FabricBlockSettings.of(Material.STONE).hardness(1.5f))
```

Comment:
Should we define hardness in a more table-driven way? I think that will help Document team.

Reply:
We can use regex to get the hardness.